v19.2/recommended-production-settings.md

```diff
@@ -23,42 +27,75 @@ Mentions of "CPU resources" refer to vCPUs, which are also known as hyperthreads
 
-- To optimize for throughput, use larger nodes, up to 32 vCPUs. Based on internal testing results, 32 vCPUs is the sweet spot for OLTP workloads. For RAM, aim for a ratio of 2 GB per vCPU.
+      {{site.data.alerts.callout_info}}
+      Note that the benefits to having more RAM decrease as the number of vCPUs increases.
+      {{site.data.alerts.end}}
 
-    To increase throughput further, add more nodes to the cluster instead of increasing node size; higher vCPUs will have [NUMA](https://en.wikipedia.org/wiki/Non-uniform_memory_access) (non-uniform memory access) implications.
+- To optimize for resiliency, use many smaller nodes instead of fewer larger nodes. Recovery from a failed node is faster when data is spread across more nodes.
 
-- To optimize for resiliency, use many smaller nodes (e.g., 4 vCPUs per node) instead of fewer larger ones. Recovery from a failed node is faster when data is spread across more nodes.
+- Avoid "burstable" or "shared-core" virtual machines that limit the load on CPU resources.
 
-- In all cases, make sure nodes are uniform to ensure consistent SQL performance.
+- To ensure consistent SQL performance, make sure all nodes have a uniform configuration.
+
+{{site.data.alerts.callout_info}}
+Underprovisioning RAM results in reduced performance (due to reduced caching and increased spilling to disk), and in some cases can cause OOM crashes. Underprovisioning CPU generally results in poor performance, and in extreme cases can lead to cluster unavailability. For more information, see [capacity planning issues](cluster-setup-troubleshooting.html#capacity-planning-issues) and [memory issues](cluster-setup-troubleshooting.html#memory-issues).
+{{site.data.alerts.end}}
 
 #### Storage
 
-- The recommended Linux filesystem is [ext4](https://ext4.wiki.kernel.org/index.php/Main_Page).
+We recommend provisioning volumes with **60 GB per vCPU**. It's fine to have less storage per vCPU if your workload does not have significant capacity needs.
 
-- Avoid using shared storage such as NFS, CIFS, and CEPH storage.
+- The maximum recommended storage capacity per node is 2 TB, regardless of the number of vCPUs.
 
-- For the best performance results, use SSD or NVMe devices. The recommended volume size is 300-500 GB.
+- Use dedicated volumes for the CockroachDB [store](architecture/storage-layer.html). Do not share the store volume with any other I/O activity.
 
-    Monitor IOPS for higher service times. If they exceed 1-5 ms, you will need to add more devices or expand the cluster to reduce the disk latency. To monitor IOPS, use tools such as `iostat` (part of `sysstat`).
+    We suggest storing CockroachDB [log files](debug-and-error-logs.html) in a separate volume from CockroachDB data so that logging is not impacted by I/O throttling. This can cause unwanted behavior on the cluster.
```

Comment:
nit/opinion: Should we elaborate on what we mean by "unwanted behavior on the cluster."

Reply:
Since this was downgraded from a 'critical' to a 'suggested' point in review, perhaps it's best not to get into it. I removed the line instead.